src/main/msp/msp.c

```diff
@@ -2074,6 +2075,31 @@ case MSP_NAME:
 #endif
         break;
 
+    case MSP2_SENSOR_CONFIG_ACTIVE:
```

Comment:
This looks much better to me.
It may be possible to keep `detectedSensors` in format expected by configurator and just dump it. But it may cause problems on refactoring.